tests/conftest.py

```diff
@@ -240,6 +247,45 @@ def minimal_attribute_dict():
     }
 
 
+@pytest.fixture()
+def minimal_dataframe_dict():
```

Comment:
S: consider renaming with minimal_dataframe_lookup?  makes more sense in line when called later to my eye.

Reply:
updated here https://github.com/lvgig/tubular/pull/253/commits/c18634432525b388cdc7c4096a606dbb60f23e29